tms_objects2.erb

```diff
@@ -490,9 +490,18 @@ xsi:schemaLocation="http://www.lido-schema.org http://www.lido-schema.org/schema
                   <lido:conceptID lido:source="<%= PCAATCN %>" lido:type="Publication related role"><%= PCAATID %></lido:conceptID>
                   <lido:term><%= pc["Role"] %></lido:term>
                 </lido:roleActor>
+        <% PubConLifeRole.each do |lr| %>
+                <lido:roleActor>
+        <% if lr["conID"] == pc["ConstituentID"] %>
+                  <lido:conceptID lido:source="<%= lr["UCTS"] %>" lido:type="<%= lr["thesrefttype"] %>"><%= lr["FinalST"] %></lido:conceptID>
+                  <lido:term><%= lr["Term"] %></lido:term>
+        <% end %>
+                </lido:roleActor>
+        <% end %>   <%# PubConLifeRole |lr| %>
+                <lido:attributionQualifierActor><%= pc["Prefix"] %></lido:attributionQualifierActor>
```

Comment:
although not in the coboat like this, it would be better to only show attributionQualifierActor when there is a value for it.